examples/embedding.c

```diff
@@ -99,6 +99,7 @@ int main()
         }
     }
 
-    jl_atexit_hook();
-    return 0;
+    int ret = 0;
+    jl_atexit_hook(ret);
```

Comment:
also wrong in build_executable, devdocs/eval.rst, devdocs/init.rst, and devdocs/embedding.rst